content/docs/user-guide/networking/settings.md

```diff
@@ -5,9 +5,41 @@ linktitle: Network Settings
 ---
 
 ## Overview
-This page documents [console variables](/docs/user-guide/appendix/cvars) and settings for AzNetworking and Multiplayer behavior.
+## Overview
+This page documents [console variables](/docs/user-guide/appendix/cvars) and other settings that can control Networking and Multiplayer behavior.
+
+## Networking commands
+These are console commands that control the flow of networked and multiplayer games, or will be required to for a networked game or a simulation.
+
+| Setting                | Description                                                                                | Notes                                                                                                                                      |
+|------------------------|--------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| host                   | Opens a multiplayer connection as a host for other clients to connect to.                  ||                                                                      ||
```

Comment:
maybe as a note for this and `disconnect` we can add a note like "Takes no arguments".

Reply:
Added params and a some notes.